config.defaults.yaml

```diff
@@ -51,3 +162,137 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
```

Comment:
We should avoid excess precision in numbers

Reply:
Where should we truncate? @mcoughlin?

Reply:
BTW, it's also untouched in this PR, so please file an issue.